client/components/contact/settings/index.js

```diff
@@ -0,0 +1,16 @@
+// Configuration settings used by the contact page
+const contactSettings = {
```

Comment:
toastSettings ?

Also, what do you think about just having this in the ContactForm.jsx? Do you think it helps to break it out into a separate file?